test/distributed/_sharded_tensor/ops/test_allclose.py

```diff
@@ -0,0 +1,51 @@
+# Owner(s): ["oncall: distributed"]
```

Comment:
Lets have a single test file for equals and allclose instead of a separate file for each.